storage: TestStoreRangeSplitRaceUninitializedRHS is disabled

SHA: https://github.com/cockroachdb/cockroach/commits/4d203e83dbc49ab1bd9283c3e15caa06e5288bb8

Stress build found a failed test:

```
I161024 06:54:28.414882 29853 storage/engine/rocksdb.go:349  opening in memory rocksdb instance
I161024 06:54:28.416494 29853 gossip/gossip.go:236  initial resolvers: []
W161024 06:54:28.416944 29853 gossip/gossip.go:1063  no resolvers found; use --join to specify a connected node
I161024 06:54:28.417342 29853 gossip/gossip.go:271  NodeID set to 1
I161024 06:54:28.445573 29853 gossip/gossip.go:271  NodeID set to 1
I161024 06:54:28.465146 29853 storage/store.go:1125  [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I161024 06:54:28.465724 29853 gossip/gossip.go:286  NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:42887" > attrs:<> locality:<> 
I161024 06:54:28.467055 29942 storage/replica_proposal.go:292  [s1,r1/1:/M{in-ax}] new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 300ms following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0s [physicalTime=1970-01-01 00:00:00 +0000 UTC]
I161024 06:54:28.480164 29853 storage/engine/rocksdb.go:349  opening in memory rocksdb instance
I161024 06:54:28.484510 29853 gossip/gossip.go:236  initial resolvers: [127.0.0.1:42887]
W161024 06:54:28.493965 29853 gossip/gossip.go:1065  no incoming or outgoing connections
I161024 06:54:28.494627 29853 gossip/gossip.go:271  NodeID set to 2
I161024 06:54:28.531071 29853 gossip/gossip.go:271  NodeID set to 2
I161024 06:54:28.532534 29853 storage/store.go:1125  [n2,s2]: failed initial metrics computation: [n2,s2]: system config not yet available
I161024 06:54:28.533088 29853 gossip/gossip.go:286  NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:38185" > attrs:<> locality:<> 
I161024 06:54:28.535397 29987 gossip/client.go:126  node 2: started gossip client to 127.0.0.1:42887
I161024 06:54:28.568458 29853 storage/replica_raftstorage.go:446  [s1,r1/1:/M{in-ax}] generated snapshot d276c9f3 for range 1 at index 13 in 152.535µs.
I161024 06:54:28.586555 29853 storage/store.go:2969  streamed snapshot: kv pairs: 32, log entries: 3
I161024 06:54:28.591220 30035 storage/replica_raftstorage.go:577  [s2] [n2,s2,r1/?:{-}]: with replicaID [?], applying preemptive snapshot at index 13 (id=d276c9f3, encoded size=16, 1 rocksdb batches, 3 log entries)
I161024 06:54:28.600861 30035 storage/replica_raftstorage.go:580  [s2] [n2,s2,r1/?:/M{in-ax}]: with replicaID [?], applied preemptive snapshot in 0.009s
I161024 06:54:28.622832 29853 storage/replica_command.go:3236  change replicas: read existing descriptor range_id:1 start_key:"" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I161024 06:54:28.709335 30041 storage/replica.go:1765  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2} for range 1: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I161024 06:54:28.767474 30024 storage/raft_transport.go:423  raft transport stream to node 1 established
I161024 06:54:29.002907 30055 storage/replica_command.go:2354  [s1,r1/1:/M{in-ax}] initiating a split of this range at key "a\x87\xff" [r2]
I161024 06:54:29.399216 29945 storage/replica_command.go:87  [s1,r1/1:/M{in-ax}] test injecting error: read at time 0.000000000,197 encountered previous write with future timestamp 0.000000000,197 within uncertainty interval
W161024 06:54:29.423410 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
I161024 06:54:29.470080 29937 storage/replica_command.go:87  [s2,r1/2:/M{in-ax}] test injecting error: read at time 0.000000000,197 encountered previous write with future timestamp 0.000000000,197 within uncertainty interval
W161024 06:54:29.549993 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.551762 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.552176 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.552583 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.648224 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.686177 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.686799 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.687649 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.693272 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.739666 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.751439 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.789087 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.821061 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.854692 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.877843 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.953516 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.953980 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
W161024 06:54:29.954412 29967 storage/store.go:2834  [s1] got error from range 2, replica {2 2 2}: raft: cannot step as peer not found
E161024 06:54:30.035843 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r1/1:{/Min-"a\x87\xff"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
E161024 06:54:30.052443 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r2/1:{"a\x87\xff"-/Max}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:54:30.064342 30117 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xff"}] initiating a split of this range at key "a\x87\xfe" [r3]
I161024 06:54:30.521781 29944 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xff"}] test injecting error: read at time 0.000000000,331 encountered previous write with future timestamp 0.000000000,331 within uncertainty interval
I161024 06:54:30.595004 29937 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xff"}] test injecting error: read at time 0.000000000,331 encountered previous write with future timestamp 0.000000000,331 within uncertainty interval
W161024 06:54:30.858533 29941 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xff"}] 1 stepped down to follower since quorum is not active
E161024 06:54:31.376471 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r3/1:"a\x87\xf{e"-f"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:54:31.380951 30135 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xfe"}] initiating a split of this range at key "a\x87\xfd" [r4]
W161024 06:54:31.637301 29972 raft/raft.go:696  [s2,r2/2:{"a\x87\xff"-/Max}] 2 stepped down to follower since quorum is not active
W161024 06:54:32.240076 29970 raft/raft.go:696  [s2,r2/2:{"a\x87\xff"-/Max}] 2 stepped down to follower since quorum is not active
W161024 06:54:32.987906 29973 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfe"}] 2 stepped down to follower since quorum is not active
W161024 06:54:33.644372 29970 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfe"}] 2 stepped down to follower since quorum is not active
I161024 06:54:34.191664 29971 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xfe"}] test injecting error: read at time 0.000000000,458 encountered previous write with future timestamp 0.000000000,458 within uncertainty interval
I161024 06:54:34.282911 29942 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xfe"}] test injecting error: read at time 0.000000000,458 encountered previous write with future timestamp 0.000000000,458 within uncertainty interval
W161024 06:54:35.443710 30196 storage/stores.go:218  range not contained in one range: [/Min,"a\x87\xfd\x00"), but have [/Min,"a\x87\xfd")
E161024 06:54:36.253696 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r4/1:"a\x87\xf{d"-e"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
W161024 06:54:36.593886 29972 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfd"}] 2 stepped down to follower since quorum is not active
E161024 06:54:37.376951 30135 internal/client/txn.go:326  [s1,r1/1:{/Min-"a\x87\xfd"}] failure aborting transaction: does not exist; abort caused by: conflict updating range descriptors
I161024 06:54:37.464997 30214 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xfd"}] initiating a split of this range at key "a\x87\xfc" [r5]
W161024 06:54:37.816936 29942 raft/raft.go:696  [s1,r3/1:"a\x87\xf{e"-f"}] 1 stepped down to follower since quorum is not active
W161024 06:54:38.140285 29937 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfd"}] 2 stepped down to follower since quorum is not active
W161024 06:54:38.940770 29971 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfd"}] 2 stepped down to follower since quorum is not active
I161024 06:54:39.338300 29970 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xfd"}] test injecting error: read at time 0.000000000,700 encountered previous write with future timestamp 0.000000000,700 within uncertainty interval
I161024 06:54:39.551955 29942 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xfd"}] test injecting error: read at time 0.000000000,700 encountered previous write with future timestamp 0.000000000,700 within uncertainty interval
W161024 06:54:39.964634 29975 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfd"}] 2 stepped down to follower since quorum is not active
E161024 06:54:41.134437 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r5/1:"a\x87\xf{c"-d"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:54:41.269392 30237 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xfc"}] initiating a split of this range at key "a\x87\xfb" [r6]
W161024 06:54:41.381339 29945 raft/raft.go:696  [s1,r5/1:"a\x87\xf{c"-d"}] 1 stepped down to follower since quorum is not active
W161024 06:54:41.389984 29976 raft/raft.go:696  [s2,r4/2:"a\x87\xf{d"-e"}] 2 stepped down to follower since quorum is not active
W161024 06:54:41.991612 29971 raft/raft.go:696  [s2,r1/2:{/Min-"a\x87\xfc"}] 2 stepped down to follower since quorum is not active
W161024 06:54:42.008226 29976 raft/raft.go:696  [s2,r4/2:"a\x87\xf{d"-e"}] 2 stepped down to follower since quorum is not active
W161024 06:54:42.758497 29941 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xfc"}] 1 stepped down to follower since quorum is not active
W161024 06:54:42.888875 29975 raft/raft.go:696  [s2,r4/2:"a\x87\xf{d"-e"}] 2 stepped down to follower since quorum is not active
W161024 06:54:43.457452 29940 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xfc"}] 1 stepped down to follower since quorum is not active
I161024 06:54:44.319333 29942 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xfc"}] test injecting error: read at time 0.000000000,853 encountered previous write with future timestamp 0.000000000,853 within uncertainty interval
I161024 06:54:44.421037 29971 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xfc"}] test injecting error: read at time 0.000000000,853 encountered previous write with future timestamp 0.000000000,853 within uncertainty interval
E161024 06:54:45.843820 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r6/1:"a\x87\xf{b"-c"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:54:45.903209 30295 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xfb"}] initiating a split of this range at key "a\x87\xfa" [r7]
W161024 06:54:47.057189 29942 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xfb"}] 1 stepped down to follower since quorum is not active
W161024 06:54:47.504827 29942 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xfb"}] 1 stepped down to follower since quorum is not active
I161024 06:54:48.432600 29941 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xfb"}] test injecting error: read at time 0.000000000,1014 encountered previous write with future timestamp 0.000000000,1014 within uncertainty interval
I161024 06:54:48.460382 29970 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xfb"}] test injecting error: read at time 0.000000000,1014 encountered previous write with future timestamp 0.000000000,1014 within uncertainty interval
E161024 06:54:49.094709 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r7/1:"a\x87\xf{a"-b"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:54:49.167712 30349 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xfa"}] initiating a split of this range at key "a\x87\xf9" [r8]
W161024 06:54:49.611844 29945 raft/raft.go:696  [s1,r6/1:"a\x87\xf{b"-c"}] 1 stepped down to follower since quorum is not active
W161024 06:54:49.752727 29943 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xfa"}] 1 stepped down to follower since quorum is not active
I161024 06:54:50.726484 29939 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xfa"}] test injecting error: read at time 0.000000000,1165 encountered previous write with future timestamp 0.000000000,1165 within uncertainty interval
I161024 06:54:50.841059 29971 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xfa"}] test injecting error: read at time 0.000000000,1165 encountered previous write with future timestamp 0.000000000,1165 within uncertainty interval
W161024 06:54:51.271284 29943 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xfa"}] 1 stepped down to follower since quorum is not active
I161024 06:54:52.310114 30361 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xf9"}] initiating a split of this range at key "a\x87\xf8" [r9]
E161024 06:54:52.361445 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r8/1:"a\x87\xf{9"-a"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
W161024 06:54:52.501981 29940 raft/raft.go:696  [s1,r8/1:"a\x87\xf{9"-a"}] 1 stepped down to follower since quorum is not active
W161024 06:54:52.995206 29945 raft/raft.go:696  [s1,r7/1:"a\x87\xf{a"-b"}] 1 stepped down to follower since quorum is not active
W161024 06:54:53.033197 29939 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
W161024 06:54:53.516195 29942 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
W161024 06:54:53.954523 29943 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
I161024 06:54:55.194546 29934 gossip/gossip.go:1079  node has connected to cluster via gossip
W161024 06:54:55.426923 29944 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
I161024 06:54:55.568042 29976 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xf9"}] test injecting error: read at time 0.000000000,1319 encountered previous write with future timestamp 0.000000000,1319 within uncertainty interval
I161024 06:54:55.601541 29940 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xf9"}] test injecting error: read at time 0.000000000,1319 encountered previous write with future timestamp 0.000000000,1319 within uncertainty interval
W161024 06:54:56.659350 29940 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
W161024 06:54:56.905768 29939 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
W161024 06:54:57.126947 29942 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
W161024 06:54:57.462471 29940 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf9"}] 1 stepped down to follower since quorum is not active
E161024 06:54:58.165295 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r9/1:"a\x87\xf{8"-9"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:54:58.258376 30423 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xf8"}] initiating a split of this range at key "a\x87\xf7" [r10]
W161024 06:54:58.676730 29946 storage/replica.go:509  [s1,r1/1:{/Min-"a\x87\xf8"}] raftMu: mutex held by github.com/cockroachdb/cockroach/pkg/storage.(*Replica).handleRaftReady for 670.057214ms (>500ms):
goroutine 29946 [running]:
runtime/debug.Stack(0x1b53d18, 0x1b53d58, 0x47)
    /usr/local/go/src/runtime/debug/stack.go:24 +0x79
github.com/cockroachdb/cockroach/pkg/util/syncutil.ThresholdLogger.func1(0x27f042fe)
    /go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/timedmutex.go:63 +0xe1
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*TimedMutex).Unlock(0xc4288071f8)
    /go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/timedmutex.go:90 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).handleRaftReady(0xc428807180, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    /go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1872 +0xee
github.com/cockroachdb/cockroach/pkg/storage.(*Store).processReady(0xc4311cac00, 0x1)
    /go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3029 +0x13e
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc426e51720, 0xc428b6fdd0)
    /go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:230 +0x28c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
    /go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc428b6fdd0, 0xc426d792e0)
    /go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
    /go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:192 +0x66
W161024 06:54:58.676954 29946 storage/store.go:3041  [s1,r1/1:{/Min-"a\x87\xf8"}] handle raft ready: 0.7s
I161024 06:54:59.730580 29946 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xf8"}] test injecting error: read at time 0.000000000,1470 encountered previous write with future timestamp 0.000000000,1470 within uncertainty interval
I161024 06:54:59.818839 29937 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xf8"}] test injecting error: read at time 0.000000000,1470 encountered previous write with future timestamp 0.000000000,1470 within uncertainty interval
W161024 06:55:00.062650 29944 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf8"}] 1 stepped down to follower since quorum is not active
W161024 06:55:00.566158 29940 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf8"}] 1 stepped down to follower since quorum is not active
E161024 06:55:01.292643 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r10/1:"a\x87\xf{7"-8"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
I161024 06:55:01.334875 30431 storage/replica_command.go:2354  [s1,r1/1:{/Min-"a\x87\xf7"}] initiating a split of this range at key "a\x87\xf6" [r11]
W161024 06:55:02.505110 29940 raft/raft.go:696  [s1,r1/1:{/Min-"a\x87\xf7"}] 1 stepped down to follower since quorum is not active
I161024 06:55:02.993515 29973 storage/replica_command.go:87  [s2,r1/2:{/Min-"a\x87\xf7"}] test injecting error: read at time 0.000000000,1603 encountered previous write with future timestamp 0.000000000,1603 within uncertainty interval
I161024 06:55:03.071633 29939 storage/replica_command.go:87  [s1,r1/1:{/Min-"a\x87\xf7"}] test injecting error: read at time 0.000000000,1603 encountered previous write with future timestamp 0.000000000,1603 within uncertainty interval
I161024 06:55:03.367859 29872 gossip/gossip.go:1079  node has connected to cluster via gossip
I161024 06:55:04.675522 30493 util/stop/stopper.go:425  quiescing; tasks left:
2      storage/client_test.go:503
1      storage/queue.go:467
1      kv/txn_coord_sender.go:911
I161024 06:55:04.676541 30493 util/stop/stopper.go:425  quiescing; tasks left:
2      storage/client_test.go:503
1      storage/queue.go:467
I161024 06:55:04.714247 30493 util/stop/stopper.go:425  quiescing; tasks left:
1      storage/queue.go:467
1      storage/client_test.go:503
E161024 06:55:04.725453 30002 storage/node_liveness.go:141  [hb] failed liveness heartbeat: failed to send RPC: sending to all 2 replicas failed; last error: failed to send RPC: store is stopped
E161024 06:55:04.732585 29917 storage/queue.go:569  [replicate] (purgatory) on [n1,s1,r11/1:"a\x87\xf{6"-7"}]: 0 of 0 stores with an attribute matching []; likely not enough nodes in cluster
E161024 06:55:04.732881 30002 storage/node_liveness.go:141  [hb] failed liveness heartbeat: node unavailable; try another peer
I161024 06:55:04.733113 30493 util/stop/stopper.go:425  quiescing; tasks left:
1      storage/queue.go:467
I161024 06:55:04.733545 30492 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
E161024 06:55:04.738313 29895 storage/node_liveness.go:141  [hb] failed liveness heartbeat: failed to send RPC: sending to all 2 replicas failed; last error: failed to send RPC: store is stopped
E161024 06:55:04.757789 29895 storage/node_liveness.go:141  [hb] failed liveness heartbeat: node unavailable; try another peer
W161024 06:55:04.771544 30026 storage/store.go:2834  [s2] got error from range 1, replica {1 1 1}: storage/raft_transport.go:249: unable to accept Raft message from {NodeID:2 StoreID:2 ReplicaID:2}: no handler registered for {NodeID:1 StoreID:1 ReplicaID:1}
W161024 06:55:04.771907 30026 storage/store.go:2834  [s2] got error from range 0, replica {1 1 0}: storage/raft_transport.go:249: unable to accept Raft message from {NodeID:2 StoreID:2 ReplicaID:0}: no handler registered for {NodeID:1 StoreID:1 ReplicaID:0}
I161024 06:55:04.774049 30492 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
I161024 06:55:04.785014 30492 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
I161024 06:55:04.786040 29903 http2_server.go:276  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:38185->127.0.0.1:57708: use of closed network connection
I161024 06:55:04.786894 29911 http2_server.go:276  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:42887->127.0.0.1:34608: use of closed network connection
I161024 06:55:04.787798 30492 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
I161024 06:55:04.788538 30492 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
    client_split_test.go:1398: storage/replica_command.go:2434: split at key "a\x87\xfd" failed: conflict updating range descriptors
```

On running `make stress PKG=./pkg/storage TESTS=TestStoreRangeSplitRaceUninitializedRHS` 
this is showing up as 

`
--- FAIL: TestStoreRangeSplitRaceUninitializedRHS (0.94s)
	client_split_test.go:1403: storage/replica_command.go:2441: split at key "a\x87\xf6" failed: result is ambiguous
`

--- a/pkg/storage/client_split_test.go
+++ b/pkg/storage/client_split_test.go
@@ -1370,7 +1370,6 @@ func TestStoreSplitTimestampCacheDifferentLeaseHolder(t *testing.T) {
 // and the uninitialized replica reacting to messages.
 func TestStoreRangeSplitRaceUninitializedRHS(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("#10172")
 	mtc := &multiTestContext{}
 	storeCfg := storage.TestStoreConfig(nil)
 	// An aggressive tick interval lets groups communicate more and thus

